### ISSUE TITLE
Two sets of templates - Back end focused (Current [End Points]) and Front end focused (Blazor Web App)

### ISSUE THREAD

**Issue by eliudfromkenya** [ardalis/CleanArchitecture#633]
Is it possible to be having 2 or 3 sets of templates so when creating the project, you can select if it's back-end, or it's front-end focused mostly using new Blazor template, or a mix of both worlds in the same project.

I have been trying to mix the two, I mean adding Blazor functionality to this template, but ...

**Comment by ardalis** [comment]
Yes, but the problem is the maintenance burden. And even with Blazor a lot of teams will want their Blazor separate from their APIs, so it would just be another net new project anyway. I do have one other template for WorkerServices but that has no real UI so it's simple. 

https://github.com/ardalis/CleanArchitecture.WorkerService

And even that one I have trouble keeping up with compared to this one.

**Comment by eliudfromkenya** [comment]
Ok! Thanks, I understand.
